gradio/external_utils.py

```diff
@@ -41,7 +46,7 @@ def get_tabular_examples(model_name: str) -> Dict[str, List[float]]:
     # replace nan with string NaN for inference API
     for data in example_data.values():
         for i, val in enumerate(data):
-            if isinstance(val, numbers.Number) and math.isnan(val):
+            if isinstance(val, float) and math.isnan(val):
```

Comment:
not possible for this to be int?

Reply:
It is but an int would never be nan so the second part of the condition would fail